public/instructors_list.php

```diff
@@ -1,15 +1,30 @@
 <?php require_once(dirname(__FILE__)."/../modules/ensure_logged_in.php"); ?>
+<?php include "templates/header.php"; ?>
 
+<?php if (get_current_role() == "admin") { ?>
 <?php
 
 require_once "../modules/models/user.php";
 require_once "../common.php";
 
 try {
-    $result = User::where(array('is_instructor' => '1'));
+    $instructors = User::where(array('is_instructor' => '1'));
+} catch (PDOException $error) {
+    echo "<br>" . $error->getMessage();
+}
+
+try {
+    $course_offering = CourseOffering::getAll();
 } catch (PDOException $error) {
-    echo $sql . "<br>" . $error->getMessage();
+    echo "<br>" . $error->getMessage();
 }
+
+try {
+    $course_assignment = CourseOfferingInstructor::getAll();
```

Comment:
Ok so you need to also preload the `course_offering` and the `user` associations here